### PR TITLE
feat: make the slot more compact

### PR DESCRIPTION
**Pull request by MrWindlike**
This PR removes the "Empty" text from the slot to make it more compact. 

And now you can drop the component into the slot content whether or not it has children which would append to the front of the slot.

image

### CODE REVIEW

File: packages/editor/src/AppModel/ComponentModel.ts [hunk 1]
```diff
@@ -183,7 +183,7 @@ export class ComponentModel implements IComponentModel {
       parent.children[slot] = [];
     }
 
-    parent.children[slot].push(this);
+    parent.children[slot].unshift(this);
```
Comment (tanbowensg) [inline packages/editor/src/AppModel/ComponentModel.ts:186]: Why use unshift? `append` usually means add to the last position.
If you want add a component before other siblings, you should first call `appendTo`, then call `moveAfter`.

**Review by tanbowensg — CHANGES_REQUESTED** [review]

File: packages/editor/src/AppModel/ComponentModel.ts (hunk 1 above)
Comment (MrWindlike) [reply packages/editor/src/AppModel/ComponentModel.ts:183]: I want to append it to the front of the slot when dropping a component into the slot item. I split the `appendTo` into `appendToRoot` and `appendToSlotFront` two functions. What do you think of this approach?

Comment (tanbowensg) [reply packages/editor/src/AppModel/ComponentModel.ts:183]: Maybe it is better to add component at the first in dropping situation, but in other situation it is not. For example, user drop component into component element in canvas. Actually, we cannot prediect whether user wants to append component to the first or the last.
There is no way to satisfy all the situations. So I think we just keep it simple and make no changes.

Comment (MrWindlike) [reply packages/editor/src/AppModel/ComponentModel.ts]: Done. I revert the related codes.

**Review by tanbowensg — APPROVED** [review]

**Pull request merged**